src/main/resources/config/application.yml

```diff
@@ -114,6 +114,8 @@ managementportal:
         keyStorePassword: radarbase
         signingKeyAlias: radarbase-managementportal-ec
         enablePublicKeyVerifiers: false
+    siteSettings:
+      hiddenSubjectFields: [person_name, date_of_birth, group]
```

Comment:
By default, this can be an empty list, since we don't want to hide these features by by default.